.github/workflows/blank.yml

```diff
@@ -0,0 +1,14 @@
+name: text-linter
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: DavidAnson/markdownlint-cli2-action@v9
+      with:
+        globs: |
+          *.md
+          !test/*.md
+      continue-on-error: true
```

Comment:
I think this is a good start. Blocking on errors, while I think the end goal of adding linting, is too much to bite off at this point. Hopefully by making these violations visible developers will see and address some as the touch files. Also this will allow us a public form to hash out rules we may want to add or exclude.